### DISCUSSION TITLE
settings export please?

### DISCUSSION THREAD

**Discussion by patrickdrd** [TrianguloY/URLCheck#288] [Ideas]
I think this app needs a settings export feature,
thanks

**Comment by TrianguloY** [comment]
Agree. And https://github.com/TrianguloY/UrlChecker/issues/175 is the request.

I would like to implement it in the next version, but I can't say when I'll be able to do it, sorry :(

**Comment by patrickdrd** [comment]
one more thing @TrianguloY , do the hosts update automatically?

**Reply by TrianguloY** [reply]
Not at the moment. That process is too long to reliably run as a background thread
